R/progress.R

```diff
@@ -24,7 +24,13 @@ progress_bar <- function(type) {
   first <- TRUE
 
   show_progress <- function(down, up) {
-    if (type == "down") {
+    if (type == "both") {
+      total <- 1
```

Comment:
Oh hmmm. Maybe it would be better to do two separate progress bars? It seems like usually either the upload or download is long, not both